code/modules/vending/modularpc.dm

```diff
@@ -0,0 +1,29 @@
+/obj/machinery/vending/modularpc
+	name = "\improper Deluxe Silicon Selections"
+	desc = "All the parts you need to build your own custom pc."
+	icon_state = "modularpc"
+	icon_deny = "modularpc-deny"
+	product_ads = "Get your gamer gear!;The best GPUs for all of your space-crypto needs!;The most robust cooling!;The finest RGB in space!"
+	vend_reply = "Game on!"
+	products = list(/obj/item/modular_computer/laptop = 4,
+					/obj/item/modular_computer/tablet = 4,
+					/obj/item/computer_hardware/hard_drive = 4,
+					/obj/item/computer_hardware/hard_drive/small = 4,
```

Comment:
why have the downgrade at the same price and with the same availability of the better one

Reply:
Well, regular-size hard drives won't fit into tablets, iirc.
Though it is still kind of silly, regardless. @Wilchenx if you want to change this, just set the custom_price var of either the small or normal drive to something more suitable.

Reply:
god i love undocumented features
![image](https://user-images.githubusercontent.com/25509881/56046436-865dd880-5d43-11e9-83ba-eb25fb1dbb2f.png)

Reply:
they say as they stare at the documentation

Reply:
can you confirm you've fixed this

Reply:
I've yet not changed the price, there's also still a merge conflict on donutstation